app/models/course.rb

```diff
@@ -174,7 +174,8 @@ def self.entry_requirement_options_without_nil_choice
   validate :validate_subject_consistency
 
   validates :name, :profpost_flag, :program_type, :qualification, :start_date, :study_mode, presence: true
-  validates :level, :age_range_in_years, presence: true, on: :create
+  validates :age_range_in_years, presence: true, unless: :further_education_course?
```

Comment:
Is dropping the `on: :create` intentional here?

Reply:
And if so, why was it there and why isn't it on `level` below. Sorry to nit-pick, but I'm just hoping you've thought about this enough to explain the change. :)

Reply:
Good spot :grimacing: I will change that now

It was explicitly on create before because we have several courses that don't have it, but we want it to be added to all future courses